internal/weldr/upload.go

```diff
@@ -112,19 +115,17 @@ func targetsToUploadResponses(targets []*target.Target, state common.ComposeStat
 		case *target.AWSTargetOptions:
 			upload.ProviderName = "aws"
 			upload.Settings = &awsUploadSettings{
-				Region:          options.Region,
-				AccessKeyID:     options.AccessKeyID,
-				SecretAccessKey: options.SecretAccessKey,
-				Bucket:          options.Bucket,
-				Key:             options.Key,
+				Region: options.Region,
+				Bucket: options.Bucket,
+				Key:    options.Key,
+				// AccessKeyID and SecretAccessKey are intentionally not included.
 			}
 			uploads = append(uploads, upload)
 		case *target.AzureTargetOptions:
 			upload.ProviderName = "azure"
 			upload.Settings = &azureUploadSettings{
-				StorageAccount:   options.StorageAccount,
-				StorageAccessKey: options.StorageAccessKey,
-				Container:        options.Container,
+				Container: options.Container,
+				// StorageAccount and StorageAccessKey is intentionally not included.
```

Comment:
```suggestion
				// StorageAccount and StorageAccessKey are intentionally not included.
```

Reply:
Hehe, as you can tell I first intended to only exclude the one ;)